ansible/list-images.yaml

```diff
@@ -1,2 +1,48 @@
 ---
-- import_playbook: _list-images.yaml
+- hosts: localhost
+  name: Setup Container
+  gather_facts: false
+
+  tasks:
+    - set_fact:
+        container_name: "{{
+          lookup('password', '/dev/null chars=ascii_letters')
+        }}"
+
+    - docker_container:
+        name: "{{ container_name }}"
+        image: docker.io/centos:8
+        auto_remove: yes
+        command: sleep 10m
+
+    - add_host:
+        name: "{{ container_name }}"
+        ansible_connection: docker
+        changed_when: false
+        ansible_python_interpreter: /usr/libexec/platform-python
+
+- hosts: "{{ hostvars['localhost'].container_name }}"
+  roles:
+    - role: packages
+    - role: version
```

Comment:
I think `images` role should be added here. 
Also I am not sure why `packages` role is required in this playbook.

Reply:
it makes sense now. Thanks for explaining it. 
w.r.t refactoring, I think there might be more cleanup needed. We can pair up offline to refactor the roles and do it as separate PR. wdyt?

Reply:
Yep sounds good.